.config/samples-config-v1.json

```diff
@@ -0,0 +1,175 @@
+                "JS",
+                "Hello World",
+                "Excel"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-HelloWorld-TaskPane-JS",
+            "onboardDate": "2024-07-01",
+            "title": "Word Hello World Task Pane Add-in",
+            "shortId": "Word Hello World",
+            "shortDescription": "A simple Word Task Pane Add-in using JavaScript.",
+            "fullDescription": "This sample demonstrates how to create a simple Word Task Pane Add-in using JavaScript.",
+            "types": [
+                "Word"
+            ],
+            "tags": [
+                "JS",
+                "Hello World",
+                "Word"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-Add-in-AIGC",
+            "onboardDate": "2024-03-15",
+            "title": "Create a Word AIGC Add-in",
+            "shortId": "Word AIGC",
+            "shortDescription": "Inserting content generated by AI or predefined into document.",
+            "fullDescription": "This Add-in shows capabilities to insert content generated by AI or predefined into document and customize the style of document.",
+            "types": [
+                "Word"
+            ],
+            "tags": [
+                "TS",
+                "AIGC",
+                "Word"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Word-Add-in-WritingAssistant",
+            "onboardDate": "2024-03-06",
+            "title": "Word Writing Assistant",
+            "shortId": "Word Writing Assistant",
+            "shortDescription": "A Word Add-in to assist writing with word document.",
+            "fullDescription": "This add-in demonstrates Word add-in capabilities to help users check errors, improve writing, and rephrase content.",
+            "types": [
+                "Word"
+            ],
+            "tags": [
+                "TS",
+                "Writing Assistant",
+                "Word"
+            ],
+            "thumbnailPath": "assets/thumbnail.png",
+            "gifPath": "assets/sampleDemo.gif",
+            "suggested": false
+        },
+        {
+            "id": "Excel-Add-in-ChartAPI-Analysis-Data",
+            "onboardDate": "2024-02-27",
+            "title": "Use the Excel Chart APIs to analyze data",
+            "shortId": "Chart API analyse data",
+            "shortDescription": "Using chart API to analyse data and showout using chart.",
+            "fullDescription": "The sample add-in demonstrates Excel add-in capablities to help users using chart API to analyse data.",
```

Comment:
Here and elsewhere, the en-US spelling is "analyze". We should standardize to that.

Reply:
Done